iris/src/iris/vector_database/lecture_unit_schema.py

```
@@ -28,11 +28,39 @@ class LectureUnitSchema(Enum):
+    RELEASE_DATE = "release_date"
+    SLIDE_VISIBILITY = "slide_visibility"
 
 
 def init_lecture_unit_schema(client: WeaviateClient) -> Collection:
     if client.collections.exists(LectureUnitSchema.COLLECTION_NAME.value):
-        return client.collections.get(LectureUnitSchema.COLLECTION_NAME.value)
+        collection = client.collections.get(LectureUnitSchema.COLLECTION_NAME.value)
+        properties = collection.config.get(simple=True).properties
+        if not any(
+            property.name == LectureUnitSchema.RELEASE_DATE.value
+            for property in properties
```

Ruff (0.15.21)

[error] Variable `property` is shadowing a Python builtin

(A001)